proto/dydxprotocol/listing/params.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package dydxprotocol.listing;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/dydxprotocol/v4-chain/protocol/x/listing/types";
+
+// ListingVaultDepositParams represents the params for PML megavault deposits
+message ListingVaultDepositParams {
```

Comment:
should we just call it `ListingParams`? in the future, there might be additional params we can add to this too

Reply:
ListingParams is a bit ambiguous since it can also refer to params associated with a single listing.
I think this is fine for now. We can alter the name in the future if its really needed